### ISSUE TITLE
(3.0Alpha) RDSetMixerPorts SLOW with Remote SQL

### ISSUE THREAD

**Issue by deltecent** [ElvishArtisan/rivendell#315]
We do a lot of remote access with Rivendell, and because we're in the sticks, we're stuck with DSL. With the increase of RD_MAX_CARDS and RD_MAX_PORTS, RDSetMixerPorts() now requires 1058 remote SQL queries. It now takes a VERY long time to startup applications like RDLibrary. So long, that RDLibrary terminates with:
```
ICE default IO error handler doing an exit(), pid = 18196, errno = 32
```
Errno 32 is "broken pipe". I don't know the details of why this would be, but I think it is now taking so long to start that something is timing out. If I comment out RDLibrary's call to RDSetMixerPorts(), it loads fine.

**Comment by ElvishArtisan** [comment]
It's basically a hangover from the 1.x days, when caed(8) didn't have MySQLaccess at all. These would certainly be better moved to caed(8) now, where they would only need to be executed once at startup.

**Comment by deltecent** [comment]
Is this something you'd like me to put into 3.0?

**Comment by ElvishArtisan** [comment]
Go for it!

**Comment by ElvishArtisan** [comment]
Fixed in c58c284.
